eox_nelp/pearson_vue/tests/test_pipeline.py

```diff
@@ -76,3 +91,362 @@ def test_get_user_data(self, phone):
+            "first_name": "John",
+            "last_name": "Doe",
+            "email": "john.doe@example.com",
+            "address": "123 Main St",
+            "city": "Anytown",
+            "country": "US",
+            "phone_number": "5551234567",
+            "phone_country_code": "1",
+            "mobile_number": "5551234567",
+            "mobile_country_code": "1",
+        }
+
+        expected_payload = {
+            "soapenv:Envelope": {
+                "soapenv:Header": {
+                    "wsse:Security": {
+                        "wsse:UsernameToken": {
+                            "wsse:Username": settings.PEARSON_RTI_WSDL_USERNAME,
+                            "wsse:Password": {
+                                "#text": settings.PEARSON_RTI_WSDL_PASSWORD
+                            },
+                        },
+                    },
+                },
+                "soapenv:Body": {
+                    "sch:cddRequest": {
+                        "@clientCandidateID": f'NELC{input_data["anonymous_user_id"]}',
+                        "@clientID": settings.PEARSON_RTI_WSDL_CLIENT_ID,
+                        "candidateName": {
+                            "firstName": input_data["first_name"],
+                            "lastName": input_data["last_name"],
+                        },
+                        "webAccountInfo": {
+                            "email": input_data["email"],
+                        },
+                        "lastUpdate": mock_timezone.now().strftime("%Y/%m/%d %H:%M:%S GMT"),
+                        "primaryAddress": {
+                            "address1": input_data["address"],
+                            "city": input_data["city"],
+                            "country": input_data["country"],
+                            "phone": {
+                                "phoneNumber": input_data["phone_number"],
+                                "phoneCountryCode": input_data["phone_country_code"],
+                            },
+                            "mobile": {
+                                "mobileNumber": input_data["mobile_number"],
+                                "mobileCountryCode": input_data["mobile_country_code"],
+                            }
+                        }
+                    },
+                },
+            },
+        }
+
+        with self.assertRaises(Exception) as context:
+            import_candidate_demographics(**input_data)
+
+        self.assertEqual(str(context.exception), "Error trying to process import candidate demographics request.")
+        mock_update_xml_with_dict.assert_called_once_with(PAYLOAD_CDD, expected_payload)
+        mock_api_client.return_value.import_candidate_demographics.assert_called_once_with(mock_update_xml_with_dict())
+
+
+class TestImportExamAuthorization(unittest.TestCase):
+    """
+    Unit tests for the import_exam_authorization function.
+    """
+
+    @patch.object(timezone, "now")
```

Comment:
Why do you have to patch timezone woo?

Reply:
Basically to compare the result with the expected output if I don't patch that the result could change due to the time execution